src/include/OSL/rendererservices.h

```diff
@@ -156,6 +450,27 @@ class OSLEXECPUBLIC RendererServices {
                                   Vec3* Pout, int npoints,
                                   TypeDesc::VECSEMANTICS vectype);
 
+    /// @brief Builds a free function to provide a value for a given attribute.
```

Comment:
Add comment that build_attribute_getter is only called during shader compilation, not execution